src/main/resources/hudson/plugins/jira/pipeline/FindIssuesAtCommitLogsStep/config.jelly

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess you mean [this](https://github.com/escoem/jira-plugin/blob/JENKINS-33996/src/main/resources/hudson/plugins/jira/JiraSite/config.jelly#L17), right?

This new jelly is defining a simplified configuration (actually just offering one single configuration option), so JiraSite jelly would not re-usable here.

This simplified jelly is only used by Pipeline Snippet Generator.